tasks/maas_setup.yml

```
@@ -24,17 +24,21 @@
   shell: |
     cd /opt/rpc-maas/playbooks
     sed -i 's|^# influx_telegraf_targets|influx_telegraf_targets|g' /etc/openstack_deploy/user_rpco_maas_variables.yml
+    date
     openstack-ansible -e maas_pre_flight_metadata_check_enabled=false site.yml -vvv
-  async: 1200
+    date
+  async: 3600
   poll: 5
```

Can we increase the polling so it doesn't dirt the log more than necessary? Is one minutes too long for the SSH timeout?

```suggestion
  poll: 60
```

---

One minute is probably too long, I would rather do 30 seconds instead of 5